pygmt/clib/session.py

```diff
@@ -1330,7 +1330,7 @@ def open_virtual_file(self, family, geometry, direction, data):
         return self.open_virtualfile(family, geometry, direction, data)
 
     @contextlib.contextmanager
-    def virtualfile_from_vectors(self, *vectors):
+    def virtualfile_from_vectors(self, vectors, *args):
```

Comment:
Also need to update the docstring at L1355 below.

```suggestion
    def virtualfile_from_vectors(self, vectors: tuple, *args):
```

Reply:
I use `Sequence` in f786b8e996aeb7c8bfc1e39fe42905fc7a000720, since a list of vectors is also allowed.